### DISCUSSION TITLE
11-28

### DISCUSSION THREAD

**Discussion by miniposi** [ChristmasTest/christmas#20] [Daily_Meeting]
### 🎅🏻 회의 내용

* 다들 개발 잘 되고 계신가요?

* 어제 한 일
  다같이 빌드 버그 고치기 !!

### ❄️ 오늘 할일

* 내일까지 해야 하는 개발 완성
* 강의 듣기

**Comment by hanseulhee** [comment]
춥 수 ... 🥶

**Comment by oauch** [comment]
오늘 날씨 짱춥 입니다 🥶
